samples/snippets/bqml_getting_started_test.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_bqml_getting_started():
+<<<<<<< HEAD
+    # [start bigquery_getting_Started_bqml_tutorial]
+    import bigframes.pandas as bpd
+=======
+    # [START bigquery_getting_Started_bqml_tutorial]
+>>>>>>> 9ec139e7e275e8082379022e3eb1ce06e9664c2e
+    from bigframes.ml.linear_model import LogisticRegression
+    import bigframes.pandas as bpd
+
+<<<<<<< HEAD
+    # Read_gbq loads a DataFrame from BiqQuery and gives an unordered, 
+    # unindexed data source. The default DataFrame will have an arbitary 
+    # index and ordering. 
```

Comment:
Let's explain why we're using `read_gbq` as well as a bit of an explanation about the use of SQL here instead of just the table names. Also link to a feature request they can watch for when they won't have to use SQL anymore: https://github.com/googleapis/python-bigquery-dataframes/issues/169

```suggestion
    # Start by selecting the data you'll use for training. `read_gbq` accepts
    # either a SQL query or a table ID. Since this example selects from multiple
    # tables via a wildcard, use SQL to define this data. Watch issue
    # https://github.com/googleapis/python-bigquery-dataframes/issues/169
    # for updates to `read_gbq` to support wildcard tables.
```

Reply:
I agree this is a better explanation of why we're using read_gbq in our code. Corrected